tests/system-tests/internal/sniffer/sniffer.go

```diff
@@ -0,0 +1,633 @@
+package sniffer
+
+import (
+	"bufio"
+	"bytes"
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/openshift-kni/eco-gotests/tests/system-tests/internal/apiobjectshelper"
+
+	"github.com/golang/glog"
+	"github.com/openshift-kni/eco-goinfra/pkg/clients"
+	"github.com/openshift-kni/eco-goinfra/pkg/deployment"
+	"github.com/openshift-kni/eco-goinfra/pkg/imagestream"
+	"github.com/openshift-kni/eco-goinfra/pkg/pod"
+	scc "github.com/openshift-kni/eco-gotests/tests/system-tests/internal/scc"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+const (
+	httpProtocol           = "http"
+	udpProtocol            = "udp"
+	snifferDeployRBACName  = "privileged-rdscore-sniffer"
+	snifferDeploySAName    = "default"
+	snifferRBACRole        = "system:openshift:scc:privileged"
+	snifferDeploymentLabel = "rds-core=sniffer-deploy"
+
+	tcpCaptureScript = `#!/bin/bash
+tcpdump -s 0 -A -v -i %s | egrep -i 'Host:'
```

Comment:
why not to narrow requests with the destination?
```suggestion
tcpdump -s 0 -A -v -i %s host %s | egrep -i 'Host:'
```